tenets/codelingo/go/naked-chan-access/example.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"time"
+	"fmt"
+)
+
+func inc(i int, c chan int) {
+    c <- i + 1
```

Comment:
this should be an issue too, right?

Reply:
I had thought that sending to a channel was fine to do outside of a select loop. Is that an anti pattern too do you think?

Reply:
well, it has some nuance, and the risk gets worse the more widely exposed the channel is – my thoughts are mostly in this:

https://github.com/juju/juju/wiki/Boring-Techniques#channels-are-awesome-but-deadly

Reply:
essentially, though – any send or receive that is not in a select is a problem; any range over a channel is also a problem. In both cases, your collaborator at the other end of the channel has the power to block you forever; if you want a robust system, you cannot afford to let that power out of your direct sight and control, or soon enough someone else will "fix a bug" in that other component and break the system.